src/services/webPlatformUtils.service.ts

```diff
@@ -269,18 +269,17 @@ export class WebPlatformUtilsService implements PlatformUtilsService {
             }
             copyEl.appendChild(textarea);
             textarea.select();
+            let success = false;
             try {
                 // Security exception may be thrown by some browsers.
-                const copyEnabled = doc.execCommand('copy');
-                if (!copyEnabled) {
-                    throw new Error('Command unsupported or disabled');
-                }
+                success = doc.execCommand('copy');
```

Comment:
In this particular case, the only cause of `execCommand` being disabled was the page being inactive. I don't know if it might be disabled by other causes - the MDN docs don't really give much info - but it didn't come up here.

I did thinking about replacing this `throw` with a `console.warn`, but then you'd garbage warnings regularly (whenever the popup is required, because the first copy will always fail).

Reply:
I think a console message is a route, just in case the failure is due to something more nefarious, it'd be nice to have a few more breadcrumbs to follow.